### DISCUSSION TITLE
[Feature] Fuzzy dates

### DISCUSSION THREAD

**Discussion by dkbarn** [immich-app/immich#12129] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Many of us store digitized scans of old photos in Immich. For these photos, the timestamp automatically extracted from EXIF data is useless, we want photos to be sorted and searchable by the date they were taken, not the date they were digitized. This requires manual override of timestamps which fortunately can be done in the app.

Unfortunately, the UI editor for timestamps requires full entry of an exact datetime: YYYY-MM-DD HH:MM:SS. For *many* of my old family photos, only the date is known, not the time, or only the month, or only the season. Or maybe I know the decade but nothing more.

In order to cater to all of these scenarios, it would be nice if we could set a date RANGE on a photo, rather than an exact timestamp. By allowing the user to set a lower and upper bound of the possible date a photo was taken, this would provide support for all the various types of uncertainty that one might encounter with photo dates.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by gaeldemarseille** [comment]
Why was this closed, I think it's a must have.I can't have all my family member have a random but chuck of picture all on a day in 2008 regardless of it it was recent or from the 60s

Also I have people who only send pictures via WhatsApp and the data is wrong. I don't day and time but I could put an estimate likely as a range. I'm not going to decide a date and specific second and claim that's the current truth. 

I don't know the database but I don't even think it that hard to make on the back side. Front is probably more annoying and common options might be prioritized I guess.

**Comment by giopaglia** [comment]
I also think it would be necessary to be able to represent some degree of uncertainty in an asset's date.

Ability to set a **datetime range** as opposed to an exact datetime looks good enough an option. As a simpler alternative, one could set a **datetime prefix**, e.g., just the year, or just the year+month, or maybe just the date.

A discussion was happening in https://github.com/immich-app/immich/discussions/6071, seems also stale though
